### DISCUSSION TITLE
Upgrading code style to PSR-12?

### DISCUSSION THREAD

**Discussion by fgaroby** [laravel/framework#43392] [Ideas] [answered]
Hi,
Is there a reason to not upgrading [the code style rule](https://laravel.com/docs/9.x/contributions#coding-style) from PSR-2 to PSR-12 ?
PSR-2 has been marked as deprecated since 2019, and Symfony uses PSR-12.

Why not upgrade this, with the next Laravel version (Laravel 10), for example?

**Comment by michael-rubel** [comment, marked as answer]
This is because the Laravel team has its own vision of code style.
If you want to follow `PSR-12` or the Laravel standard, you can use [`Laravel Pint`](https://github.com/laravel/pint) tool.

---
